src/ser/error.rs

```diff
@@ -37,6 +40,9 @@ impl fmt::Display for Error {
         match *self {
             Error::Io(ref inner) => inner.fmt(fmt),
             Error::InvalidDocumentKey(ref key) => write!(fmt, "Invalid map key type: {}", key),
+            Error::InvalidCString(ref string) => {
+                write!(fmt, "cstrings cannot contain null bytes: {}", string)
```

Comment:
Minor: might want to use `{:?}` instead to make it easier to see the escaped contents of the string.

Reply:
good idea, updated